### DISCUSSION TITLE
Unsure why application rejected

### DISCUSSION THREAD

**Discussion by MishaBTW** [community/community#140463] [GitHub Education]
### Hi everyone,

I've been trying to apply for GitHub Education. I went through and addressed the reasons for rejection, adding billing information, my student e-mail address, etc. to my account. After resubmitting my application this most recent time I received a rejection again, but it gave no particular reason why it was rejected. After looking through the pages for common reasons for rejection, I'm not sure what to do, as I'm not sure why I was rejected. One thing that seemed strange to me was previously one reason I was given is that my student ID did not display the school's full name on it. However, that's not true. My ID does contain the school's full name on it and it is clearly visible in all of the photos of my student ID that I have submitted. I'm not sure if that is related to my most recent rejection or not, as it wasn't listed as a reason this time (nothing in particular was). Any suggestions as to what I might do? I'm at a loss as to what to do next as I wasn't given a reason for rejection.

**Comment by KavinMadusanka** [comment]
Double-check your submission:
Ensure that your student ID clearly shows the full name of your institution, the expiration date, and any other required information.
Verify that your email address is associated with your educational institution and that all necessary documentation (like proof of enrollment) is current and legible.
Make sure the billing information provided matches your student status and doesn't have inconsistencies.

Review Common Rejection Reasons Again:
Even though you’ve already checked the common rejection reasons, go through them one more time carefully. Sometimes, a minor detail might be overlooked, or something in your documents might not be as clear as you think.

Contact GitHub Support:
Since no specific reason was given in your most recent rejection, it’s a good idea to reach out directly to GitHub Support. Politely explain your situation, mention the previous rejection reason (regarding the student ID), and request clarification or guidance on what you can improve. You can use GitHub’s support form or their email for educational support.

Resubmit with Additional Clarification:
In your next submission, include a short explanation about the issue with your student ID and make it very clear that the full name of your school is visible. You might want to attach a new photo of your ID and proof of enrollment (if applicable).
If possible, attach additional documentation that proves you are a student.
Double-check your school's inclusion:

Confirm that your institution is on the list of recognized schools in the GitHub Education program. Sometimes, institutions may not be automatically recognized by GitHub, which could cause an issue.
